proposals/description/fip_28.json

```diff
@@ -0,0 +1,87 @@
+{
+    "proposal_title": "FIP-28/29 FeiRari TRIBE and Optimistic Approval",
+    "proposal_commands": [
+        {
+            "target": "rariPool8Comptroller",
+            "values": "0",
+            "method": "_setBorrowPaused(address,bool)",
+            "arguments": [
+                "0xFd3300A9a74b3250F1b2AbC12B47611171910b07",
+                true
+            ],
+            "description": "Make fTRIBE unborrowable"
+        },
+        {
+            "target": "fei",
+            "values": "0",
+            "method": "mint(address,uint256)",
+            "arguments": [
+                "0xbC9C084a12678ef5B516561df902fdc426d95483",
```

Comment:
this address is a timelock, not a multisig

Reply:
That is correct. We don't want to give 50M to the multisig